changelogs/api-management-service.md

```diff
@@ -1,5 +1,38 @@
 # Azure API Management service changelog
 
+## Release - API Management service: June, 2024
+
+###  ‼️ Breaking changes ‼️
+
+- URL properties in the [Backend](https://learn.microsoft.com/rest/api/apimanagement/backend/create-or-update) entity no longer accept URLs containing query parameters. We've made the change to fix a bug. Please use [`set-query-parameter`](https://learn.microsoft.com/azure/api-management/set-query-parameter-policy) policy and [`backend.credentials.query`](https://learn.microsoft.com/rest/api/apimanagement/backend/create-or-update#backendcredentialscontract) to set query parameters.
+
+### New features, improvements, and changes
+
+- HTTP/2 connections are now drained and gracefully terminated before node restarts
+- We added support for decryption keys to the [`validate-azure-ad-token`](https://learn.microsoft.com/azure/api-management/validate-jwt-policy) policy
```

Comment:
The correct link is https://learn.microsoft.com/azure/api-management/validate-azure-ad-token-policy